tests/integration/utils.py

```diff
@@ -290,36 +296,36 @@ def build_image_and_check_cmd(
     :param cachi2_image: ContainerImage instance with Cachi2 image
     """
     log.info("Create cachi2.env file")
-    env_vars_file = os.path.join(tmpdir, "cachi2.env")
+    env_vars_file = tmp_path.joinpath("cachi2.env")
     cmd = [
         "generate-env",
         output_folder,
         "--output",
         env_vars_file,
         "--for-output-dir",
-        os.path.join("/tmp", f"{test_case}-output"),
+        str(Path("/tmp").joinpath(f"{test_case}-output")),
```

Comment:
I think we don't need to convert to string here - the `output_folder` and `env_vars_file` above are Paths as well, and it works